.travis.yml

```diff
@@ -41,7 +41,9 @@ jobs:
     - env: TOXENV=py36-freeze
       python: '3.6'
     - env: TOXENV=py37
-      python: 'nightly'
+      python: 3.7
```

Comment:
we should probably put quotes around this at some point

Reply:
Oh right, classical YAML snafu. I will do it on `master` after we release this. 👍

Reply:
Done! 😉

Reply:
What is the reasoning behind this (out of curiosity)?
Is Travis gently handling the float here?

Reply:
travis allows both float and string here and yaml is funtimes